ajax/libs/datalist-polyfill/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "datalist-polyfill",
+  "description": "A minimal and dependency-free vanilla JavaScript datalist polyfill. Supports all standard's functionality as well as mimics other browsers behavior.",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/mfranzke/datalist-polyfill.git"
+  },
+  "author": "Maximilian Franzke",
+  "license": "MIT",
+  "filename": "datalist-polyfill.min.js",
+  "keywords": [
+    "datalist",
+    "polyfill",
+    "javascript",
+    "html",
+    "autosuggest",
+    "autosuggester",
+    "suggest",
+    "suggester",
+    "select"
+  ],
+  "npmName": "datalist-polyfill",
+  "npmFileMap": [
```

Comment:
Why do you use npm? On Git number of releases more than on npm.

Reply:
It‘s mainly about two of the earliest releases, that included both a typo and adding the package.json for npm usage - those wouldn‘t be necessary to reference any more, as they are way behind regarding functionality and bugfixes.

And as you seem to favor npm over github I‘ve than chosen that one to inlcude for auto-update.